src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.sql.Timestamp
+import java.util.UUID
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.model._
+
+/**
+ * Created by dvoet on 2/9/16.
+ */
+class AttributeComponentSpec extends TestDriverComponent with AttributeComponent {
+  import driver.api._
+
+  "AttributeComponent" should "insert string attribute" in {
+    val testAttribute = AttributeString("test")
+    val insertedIds = attributeQuery.insertAttributeRecords("test", testAttribute, UUID.randomUUID()).map(x => runAndWait(x))
```

Comment:
These can be `map(runAndWait(_))` but I don't care enough to ask for this change.  Do it if you like.